client/src/components/misc/LoadingAnimation/LoadingAnimation.scss

```diff
@@ -0,0 +1,49 @@
+.loading-animation-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.loading-animation {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  $size: 28vmin;
+  animation: 0.9s linear infinite spinner;
+  animation-play-state: inherit;
+  border: solid 5px var(--light-purple) {
+    bottom-color: var(--purple);
+    radius: 50%;
+  }
+  height: $size;
+  width: $size;
+}
+
+@keyframes spinner {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.logo {
+  position: absolute;
+  height: 25vmin;
+}
+
+/* Remove everything under this to stop logo spinning */
```

Comment:
yep, just remove the spinning logo part! it might just be me, but I get dizzy looking at it 😂